stdlib/Distributed/src/managers.jl

```diff
@@ -592,12 +592,22 @@ It should cause the remote worker specified by `pid` to exit.
 on `pid`.
 """
 function kill(manager::ClusterManager, pid::Int, config::WorkerConfig)
-    remote_do(exit, pid)
+    try
+        remote_do(exit, pid) # For TCP based transports this will result in a close of the socket
+                           # at our end, which will result in a cleanup of the worker.
+    catch ex
+        @warn("Failed to kill remote worker $pid. Worker unreachable, unresponsive or already exited.")
```

Comment:
Discarding ex is probably a bad idea

Reply:
Yes, since we have the information, it's a good idea to add it to the warning message.

Reply:
done in https://github.com/JuliaLang/julia/pull/32290/commits/cfefd407bbc820d44eaca6b1e98e9474f2fe68f6